core/src/main/java/tech/tablesaw/joining/DataFrameJoiner.java

```diff
@@ -59,12 +74,20 @@ public Table inner(Table... tables) {
      */
     public Table inner(boolean allowDuplicateColumnNames, Table... tables) {
         Table joined = table;
-        for (Table table2 : tables) {
-          joined = inner(table2, column.name(), allowDuplicateColumnNames);
+        
+        for (int i=0; i<tables.length; i++) {
```

Comment:
can you add a space before and after the `=` and `<` for consistency with the rest of the code?